primitives/assert.futil

```diff
@@ -0,0 +1,8 @@
+
+extern "assert.sv" {
+   primitive assert_prim(@data in: 1, en: 1, @clk clk:1, @reset reset: 1) -> (out: 1);
```

Comment:
You can use the [`@protected`](https://docs.calyxir.org/lang/attributes.html#protected) attribute when you declare the submodule. This is not quite the best solution because you want to _always_ add the attribute for this cell.